### PR TITLE
Doc Updates Related to Source Tracked Org Changes

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
These doc changes reflect Org browser availability for source tracked orgs and push-pull on source-tracked sandboxes.

### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @W-12779269@

### Functionality Before
Org browser documented as being unavailable for source tracked org. Push-pull commands only documented for scratch org and not for source-tracked sandboxes. Before, deploy-retrieve commands were not available for scratch orgs and source tracked sandboxes.
We also consolidated metadata commands into a new topic called "metadata operations".

### Functionality After
Documentation and commands are updated.

### CODE REVIEW

File: docs/_articles/en/user-guide/detect-conflicts.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ lang: en
 ---
 ## Overview
 
-When you deploy source to your default org, the operation could detect conflicts on deploy to help you avoid accidentally overwriting others’ changes in the Org. The conflict detection feature is available for any org that is not using source tracking, which is most non-scratch orgs such as sandboxes or dev orgs.
+When you deploy source to your default org, the operation could detect conflicts on deploy to help you avoid accidentally overwriting others’ changes in the Org. The conflict detection feature can be turned on for any org that is not using source tracking, which is most non-scratch orgs such as sandboxes or dev orgs. Note that this feature is always on for scratch orgs and source tracked sandboxes.
```
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/detect-conflicts.md:7]: 👍🏽

File: docs/_articles/en/user-guide/manifestbuilder.md [hunk 2]
```diff
@@ -56,7 +55,7 @@ You can retrieve source for a manifest, source files, directories, or folders:
 
 When you select an item to retrieve source, only the existing nested items in the directory structure are retrieved. For example, if you retrieve source for the `classes` folder, the Apex classes that **currently exist in that directory** are retrieved. The command doesn’t retrieve all the Apex classes in the org; it only updates the classes that already exist in the folder. If you want to retrieve a new Apex class, add that class (or all Apex classes) to a `package.xml` file and retrieve source using the manifest file. You could also use a terminal to run `sfdx force:source:retrieve --metadata ApexClass:YourApexClass`.
 
-You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retrive source for non-source-tracked orgs.
+You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retrive source.
```
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/manifestbuilder.md:58]:
```suggestion
You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retreive source.
```

File: docs/_articles/ja/codebuilder/cb-start.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ The Code Builder user interface is divided into five main areas.
 **Status Bar**: Shows information about the opened project and the files you’re editing. The Status Bar displays the org that you're connected to. 
 
 ## Get To Know Your Org – Use the Org Browser
-Take a closer look at the activity bar and notice a cloud icon. This icon represents the Org Browser. It’s a part of the Salesforce Extension pack. It helps you browse and retrieve metadata from your org without having to use a manifest file. The Org Browser is available only in non source-tracked orgs such as sandboxes or Developer Edition orgs. User Org Browser to retrieve the metadata source into your local project. 
+Take a closer look at the activity bar and notice a cloud icon. This icon represents the Org Browser. It’s a part of the Salesforce Extension pack. It helps you browse and retrieve metadata from your org without having to use a manifest file. Use the Org Browser to deploy or retrieve metadata.
```
Comment (klewis-sfdc) [inline docs/_articles/ja/codebuilder/cb-start.md:60]: I don't think you can deploy from Org Browser - IIRC it uses retrieve only.

File: docs/_articles/ja/user-guide/manifestbuilder.md [hunk 4]
```diff
@@ -56,7 +54,7 @@ You can retrieve source for a manifest, source files, directories, or folders:
 
 When you select an item to retrieve source, only the existing nested items in the directory structure are retrieved. For example, if you retrieve source for the `classes` folder, the Apex classes that **currently exist in that directory** are retrieved. The command doesn’t retrieve all the Apex classes in the org; it only updates the classes that already exist in the folder. If you want to retrieve a new Apex class, add that class (or all Apex classes) to a `package.xml` file and retrieve source using the manifest file. You could also use a terminal to run `sfdx force:source:retrieve --metadata ApexClass:YourApexClass`.
 
-You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retrive source for non-source-tracked orgs.
+You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retrive source from orgs.
```
Comment (klewis-sfdc) [inline docs/_articles/ja/user-guide/manifestbuilder.md:57]:
```suggestion
You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retreive source from orgs.
```

**Review by klewis-sfdc — COMMENTED** [review]
Looking good, thank you @sbudhirajadoc !  I left a few comments and suggestions for consideration.

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+### Usage
+Menu options `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` appear when you right-click a file or folder.  
+
+![Source Diff command](../../../images/source_diff.png)
+
+You can then view individual file diffs side by side. If a file doesn’t exist locally or in the Org, it will not show up in the list.
+
+## Retrieve Source
+
+Org Development model doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files. When you retrieve source from an org, you could enable conflict detection between the org and the local metadata. See [Detect Conflicts](./en/user-guide/detect-conflicts).
```
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/metadata-operations.md:18]: Conflict detection is deploy-only so this should prob be removed:

> When you retrieve source from an org, you could enable conflict detection between the org and the local metadata. See [Detect Conflicts](./en/user-guide/detect-conflicts).

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+- With a source file open in the editor, right-click in the editing pane and select **SFDX: Deploy This Source File to Org**.
+- With a source file open in the editor, open the command palette and run **SFDX: Deploy This Source File to Org**.
+
+## Delete Source
+
+You can delete source from your project and from your org.
+
+- In the VS Code explorer, right-click a source file or a directory and select **SFDX: Delete from Project and Org**.
+- With a source file open in the editor, right-click the file and select **SFDX: Delete This from Project and Org**.
+- With a source file open in the editor, open the Command Palette and run **SFDX: Delete from Project and Org**.
```
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/metadata-operations.md:77]:
```suggestion
- With a source file open in the editor, open the Command Palette and run **SFDX: Delete This from Project and Org**.
```

File: docs/_articles/en/user-guide/detect-conflicts.md (hunk 1 above)
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/detect-conflicts.md:7]: I've seen and written "non-source-tracked", "non-source tracked" and "non source-tracked".  Which one should be settle on? :)

Comment (klewis-sfdc) [reply docs/_articles/en/user-guide/detect-conflicts.md:7]: Oh, I see.  Here we say non-scratch org.  Ok.  And above you have "non source-tracked".  👍🏽

**Review by klewis-sfdc — COMMENTED** [review]
This looks great!  One sentence to remove I think, re: retrieve and conflict detection, but otherwise this is 👍🏽.  Love these docs updates.

File: docs/_articles/en/user-guide/metadata-operations.md (hunk 5 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/user-guide/metadata-operations.md]: Done. Thanks for catching that.

**Review by klewis-sfdc — APPROVED** [review]
LGTM!

File: docs/_articles/en/user-guide/deploy-on-save.md [hunk 7]
```diff
@@ -31,7 +31,7 @@ When you enable deploy on save for your project:
 
 - If there isn’t an active deployment and no files are queued for deployment, a file save triggers an immediate deployment.
 
-If you enable deploy on save while working against a scratch org, the context of the extension is package development model. This means that every time you save a file, **SFDX: Push Source to Default Scratch Org** is initiated and runs `force:source:push` under the hood. In org development model that works with non-source-tracked-orgs, every file save initiates **SFDX: Deploy Source to Org**, which runs `force:source:deploy`.
+If you enable deploy on save while working against a source-tracked org, every time you save a file, **SFDX: Push Source to Default Org** is initiated and runs `force:source:push` under the hood. For non source-tracked orgs, every file save initiates **SFDX: Deploy Source to Org**, which runs `force:source:deploy`.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/deploy-on-save.md:34]: @sbudhirajadoc - are the palette menu options still using the sfdx commands?

File: docs/_articles/en/user-guide/detect-conflicts.md (hunk 1 above)
Comment (ekapner) [inline docs/_articles/en/user-guide/detect-conflicts.md:7]:
```suggestion
When you deploy source to your default org, the operation could detect conflicts on deploy to help you avoid accidentally overwriting others’ changes in the org. The conflict detection feature can be turned on for any org that is not using source tracking, which is most non-scratch orgs such as sandboxes or dev orgs. Note that this feature is always on for scratch orgs and source tracked sandboxes.
```

File: docs/_articles/en/user-guide/manifestbuilder.md [hunk 8]
```diff
@@ -5,17 +5,16 @@ lang: en
...
 ## Create Project with Manifest
 
-To create developing against non-source-tracked orgs:
 
 1. Open the VS Code editor and from the Command Palette, run **SFDX: Create Project with Manifest**.
 
 ![Create project](./images/create-project-with-manifest.png)
 
-1. In the code editor's status bar, click Org Picker to open the Command Palette. You can select from the list of authorized orgs, or you can choose to authorize a new org.
+2. In the code editor's status bar, click Org Picker to open the Command Palette. You can select from the list of authorized orgs, or you can choose to authorize a new org.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/manifestbuilder.md:13]:
```suggestion
2. In the code editor's status bar, click Org Picker to open the Command Palette. Select from the list of authorized orgs, or choose to authorize a new org.
```

File: docs/_articles/en/user-guide/manifestbuilder.md [hunk 9]
```diff
@@ -5,17 +5,16 @@ lang: en
...
 
 If this is the first time you are creating a project in VS Code and haven’t authorized an org, the display text for the Org Picker shows No Default Org Set.
 
-1. Run **SFDX: Authorize an Org** and select a login URL, for example Sandbox.
+3. Run **SFDX: Authorize an Org** and select a login URL, for example Sandbox.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/manifestbuilder.md:17]:
```suggestion
3. Run **SFDX: Authorize an Org** and select a login URL, for example, Sandbox.
```

File: docs/_articles/en/user-guide/manifestbuilder.md [hunk 10]
```diff
@@ -35,69 +34,4 @@ Automatically generate a manifest file for a given a set of metadata components
...
-
-### Delete Source
-
-You can delete source from your project and from your non-source-tracked org.
-
-- In the VS Code explorer, right-click a manifest, a source file, or a directory and select **SFDX: Delete from Project and Org**.
-- With a source file open in the editor, right-click the file and select **SFDX: Delete This from Project and Org**.
-- With a source file open in the editor, open the Command Palette and run **SFDX: Delete from Project and Org**.
\ No newline at end of file
+Use the new manifest file to deploy and retrieve source from the org using **SFDX: Retrieve Source in Manifest from Org** or **SFDX: Retrieve Source in Manifest from Org** commands.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/manifestbuilder.md:37]:
```suggestion
Use the new manifest file to deploy and retrieve source from the org using **SFDX: Retrieve Source in Manifest from Org** or **SFDX: Retrieve Source in Manifest from Org** command.
```

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
+---
+title: Metadata Operations
+lang: en
+---
+
+## Source Diff
+The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at diffs for an individual file or directory.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/metadata-operations.md:7]: "diff metadata types" is a bit slangy. I suggest you tighten the whole thing up. Ex:
The Source Diff command makes it easy to visual the changes between files in your local project and the metadata in your org. You can look at diffs...

File: docs/_articles/en/user-guide/deploy-on-save.md (hunk 7 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/user-guide/deploy-on-save.md:34]: yes they are...

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+### Usage
+Menu options `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` appear when you right-click a file or folder.  
+
+![Source Diff command](../../../images/source_diff.png)
+
+You can then view individual file diffs side by side. If a file doesn’t exist locally or in the Org, it will not show up in the list.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/metadata-operations.md:14]:
```suggestion
You can then view individual file diffs side-by-side. If a file doesn’t exist locally or in the org, it doesn't show up in the list.
```

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+## Retrieve Source
+
+Org Development model doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/metadata-operations.md:18]: I would substitute "org development model" with what's truly going on. The "model" itself doesn't do anything. So are you trying to say?
If you're using a non-source-tracked org, VS Code doesn't automatically track changes, so you have to track the changes you retrieve manually.

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+  - With a manifest file open in the editor, open the Command Palette and run **SFDX: Retrieve Source in Manifest from Org**.
+- Source File or Directory
+  - In VS Code explorer, right-click a source file or a directory and select **SFDX: Retrieve Source from Org**.
+  - With a source file open in the editor, right-click in the editing pane and select **SFDX: Retrieve This Source from Org**.
+  - With a source file open in the editor, open the Command Palette and run **SFDX: Retrieve This Source from Org**.
+
+When you select an item to retrieve source, only the existing nested items in the directory structure are retrieved. For example, if you retrieve source for the `classes` folder, the Apex classes that **currently exist in that directory** are retrieved. The command doesn’t retrieve all the Apex classes in the org; it only updates the classes that already exist in the folder. If you want to retrieve a new Apex class, add that class (or all Apex classes) to a `package.xml` file and retrieve source using the manifest file. You could also use a terminal to run `sfdx force:source:retrieve --metadata ApexClass:YourApexClass`.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/metadata-operations.md:32]:
```suggestion
When you select an item to retrieve source, only the existing nested items in the directory structure are retrieved. For example, if you retrieve source for the `classes` folder, the Apex classes that **currently exist in that directory** are retrieved. The command doesn’t retrieve all the Apex classes in the org; it updates only the classes that already exist in the folder. To retrieve a new Apex class, add that class (or all Apex classes) to a `package.xml` file and retrieve the source using the manifest file. You could also use a terminal to run `sfdx force:source:retrieve --metadata ApexClass:YourApexClass`.
```

File: docs/_articles/en/user-guide/metadata-operations.md (hunk 13 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/user-guide/metadata-operations.md:18]: great point! thanks! I'l make that update.

File: docs/_articles/ja/user-guide/manifestbuilder.md [hunk 15]
```diff
@@ -56,7 +54,7 @@ You can retrieve source for a manifest, source files, directories, or folders:
 
 When you select an item to retrieve source, only the existing nested items in the directory structure are retrieved. For example, if you retrieve source for the `classes` folder, the Apex classes that **currently exist in that directory** are retrieved. The command doesn’t retrieve all the Apex classes in the org; it only updates the classes that already exist in the folder. If you want to retrieve a new Apex class, add that class (or all Apex classes) to a `package.xml` file and retrieve source using the manifest file. You could also use a terminal to run `sfdx force:source:retrieve --metadata ApexClass:YourApexClass`.
 
-You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retrive source for non-source-tracked orgs.
+You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retreive source from orgs.
```
Comment (ekapner) [inline docs/_articles/ja/user-guide/manifestbuilder.md:57]:
```suggestion
You can also use [Org Browser](./en/user-guide/development-models/#create-project-and-use-org-browser) to retrieve source from orgs.
```

**Review by ekapner — COMMENTED** [review]
Some minor editorial feedback.

File: docs/_articles/en/user-guide/deploy-on-save.md (hunk 7 above)
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/deploy-on-save.md:34]:
```suggestion
If you enable deploy on save while working against a source-tracked org, every time you save a file, **SFDX: Push Source to Default Org** is initiated and runs `force:source:push` under the hood. For non-source-tracked orgs, every file save initiates **SFDX: Deploy Source to Org**, which runs `force:source:deploy`.
```

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+### Usage
+Menu options `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` appear when you right-click a file or folder.  
+
+![Source Diff command](../../../images/source_diff.png)
+
+You can then view individual file diffs side-by-side. If a file doesn’t exist locally or in the org, it doesn't show up in the list.
+
+## Retrieve Source
+
+For non-source-tracked orgs, VS Code doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files unless you have conflict detection turned on. See [Detect Conflicts](./en/user-guide/detect-conflicts).
```
Comment (klewis-sfdc) [inline docs/_articles/en/user-guide/metadata-operations.md:18]: There is no conflict detection on retrieve operations.  There is currently a suggestion in the docs to run the diff commands before retrieving.  

```suggestion
For non-source-tracked orgs, VS Code doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files.
```

File: docs/_articles/en/user-guide/metadata-operations.md [hunk 17]
```diff
@@ -0,0 +1,90 @@
+---
+title: Metadata Operations
+lang: en
+---
+## Overview
+
+
+Say you want to change an existing Apex class or add a custom object to your production org. You first make and test these changes in a scratch org, check for conflicts, and then deploy the changes to another org for your team to access them. This ability to move changes between different Salesforce development environments such as scratch orgs and sandboxes, for various stages of testing before launching into production, is key to Salesforce development. Use these metadata operations to deploy your local changes to an org as a part of your ALM process. With conflict detection turned on, you don't have to worry about accidentally overwriting others' changes in the org. You can track changes by easily viewing changes between your local files and your default org at any time. If you’re working in a team, you can retrieve the latest changes in your org on to your local Salesforce project.
```
Comment (AnanyaJha) [inline docs/_articles/en/user-guide/metadata-operations.md:8]:
```suggestion
This ability to move changes between different Salesforce development environments such as scratch orgs and sandboxes, for various stages of testing before launching into production, is key to Salesforce development. 
```
minor nit - this sentence has a ton of great info that's important to highlight, but it's a bit long so I'm getting a little lost. Could we rephrase or break it up a bit?

Comment (AnanyaJha) [reply docs/_articles/en/user-guide/metadata-operations.md:8]: oops not suggesting we delete the other sentences lol

**Pull request merged**